### ISSUE TITLE
Code should read an IDL sav CUBE file from Cube Generator

### ISSUE THREAD

**Issue by spacemanjosh** [Cassini-UVIS/fits_writer#3]
Currently the fits_writer code only supports the IDL fits output files from Cube Generator.  But it would be easy enough to use the IDL sav file reader from scipy.

**Comment by michaelaye** [comment]
we actually tried that with a few files and it doesn't seem to be straight forward due to deep nesting and byte order flipping. do you have a positive example that for sure works?
I think we might also simply say that everybody should use the new PDS4 format, as the IDL sav files aren't publicly available, correct?

**Comment by spacemanjosh** [comment]
That's true, I think now this may no longer be needed.  I was originally thinking we might need to read the IDL save files.  In addition, along the way, I wrote an IDL function that converts the save file to FITS within IDL, so the user can just use that if need be (which is the same function I added to Cube Generator to produce FITS output from there).  
Note that the CG FITS files are not PDS4 compliant.  They become inputs into the `CGCubeToFITS` class which then creates the PDS4 compliant files.
In any case we can probably close out this ticket.

**Comment by michaelaye** [comment]
ok, it's your ticket, feel free to close.
